src/modules/tools/endstops/Endstops.cpp

```diff
@@ -1085,6 +1098,35 @@ void Endstops::on_gcode_received(void *argument)
                 gcode->stream->printf(" will take effect next home\n");
                 break;
 
+            case 211: { //Enable or disable endstop(s) hard limit
+                if(gcode->has_letter('S')) { //Alter endstop behavior    
+                        for(auto& e : endstops) {
+                            if(gcode->has_letter(e->axis)) {
+                                if(e->is_max_stop == gcode->get_value(e->axis)) {  //Are we configuring a min or max endstop?
```

Comment:
is_max_stop is a bool it is not good practice to compare with a float. also I don;t see how this selects a min or max endstop.

Reply:
That was my mistake, I previously tested min or max via:  "gcode->get_value(e->axis)!=0" to convert to a bool, not sure when I undid that...

Fixing now, will wait to hear if there's a better way to determine min/max location of the stop before any further commits.